react/src/lib/components/DeckGLMap/layers/wells/wellsLayer.ts

```diff
@@ -70,10 +71,12 @@ export default class WellsLayer extends CompositeLayer<
     }
 
     renderLayers(): (GeoJsonLayer<Feature> | PathLayer<LogCurveDataType>)[] {
+        const data = splineRefine(this.props.data);
```

Comment:
Maybe add a prop to toggle spline interpolation on and off (default on), in case the trajectories have enough sample points or for performance reasons.